src/l1-resources/search-index/index.ts

```diff
@@ -284,6 +300,81 @@ export function toJson_ApiAtlasFtsSynonymMappingDefinitionView(
 }
 /* eslint-enable max-len, quote-props */
 
+/**
+ * Tokenizer that you want to use to create tokens. Tokens determine how Atlas Search splits up text into discrete chunks for indexing.
+ *
+ * @schema ApiAtlasFTSAnalyzersTokenizer
+ */
+export interface ApiAtlasFtsAnalyzersTokenizer {
+  /**
+   * Characters to include in the longest token that Atlas Search creates.
+   *
+   * @schema ApiAtlasFTSAnalyzersTokenizer#MaxGram
+   */
+  readonly maxGram?: number;
+
+  /**
+   * Characters to include in the shortest token that Atlas Search creates.
+   *
+   * @schema ApiAtlasFTSAnalyzersTokenizer#MinGram
+   */
+  readonly minGram?: number;
+
+  /**
+   * Human-readable label that identifies this tokenizer type.
+   *
+   * @schema ApiAtlasFTSAnalyzersTokenizer#Type
+   */
+  readonly type?: string;
+
+  /**
+   * Index of the character group within the matching expression to extract into tokens. Use `0` to extract all character groups.
+   *
+   * @schema ApiAtlasFTSAnalyzersTokenizer#Group
+   */
+  readonly group?: number;
+
+  /**
+   * Regular expression to match against.
+   *
+   * @schema ApiAtlasFTSAnalyzersTokenizer#Pattern
+   */
+  readonly pattern?: string;
+
+  /**
+   * Maximum number of characters in a single token. Tokens greater than this length are split at this length into multiple tokens.
+   *
+   * @schema ApiAtlasFTSAnalyzersTokenizer#MaxTokenLength
+   */
+  readonly maxTokenLength?: number;
+}
+
+/**
+ * Converts an object of type 'ApiAtlasFtsAnalyzersTokenizer' to JSON representation.
+ */
+/* eslint-disable max-len, quote-props */
+export function toJson_ApiAtlasFtsAnalyzersTokenizer(
+  obj: ApiAtlasFtsAnalyzersTokenizer | undefined
+): Record<string, any> | undefined {
+  if (obj === undefined) {
+    return undefined;
+  }
+  const result = {
+    MaxGram: obj.maxGram,
+    MinGram: obj.minGram,
+    Type: obj.type,
+    Group: obj.group,
+    Pattern: obj.pattern,
+    MaxTokenLength: obj.maxTokenLength,
+  };
+  // filter undefined values
+  return Object.entries(result).reduce(
+    (r, i) => (i[1] === undefined ? r : { ...r, [i[0]]: i[1] }),
+    {}
+  );
+}
+/* eslint-enable max-len, quote-props */
+
```

Comment:
is it worth adding a test under `test/l1-resources/search-index`?

Reply:
thx, added